static/js/controllers/contacts-edit.js

```diff
@@ -1,4 +1,5 @@
-var _ = require('underscore');
+var _ = require('underscore'),
+    uuidV1 = require('uuid/v1');
```

Comment:
Not a particularly conscious one no. I picked it over v4 because v1 is arguably less likely to conflict (since it's time-based). However, since the chance of v4 conflicting is essentially 0, it doesn't really matter.

To reduce surprise I'll make it v4